library/tasks/CreateCountMatrix.wdl

```diff
@@ -80,10 +80,35 @@ task CreateSparseCountMatrix {
   command {
     set -e
 
+    if [[ "${gtf_file}" =~ \.gz$ ]]; then
```

Comment:
I'd suggest using `file` to check whether it's gzipped instead of relying on the extension. See e.g. https://stackoverflow.com/questions/18278990/easiest-way-to-check-for-file-extension-in-bash

Reply:
Changed